src/lib/theme/_overrides.scss

```diff
@@ -224,6 +224,48 @@ $shape-radius: 8px;
       height: 3px;
       border-radius: 3px 3px 0 0;
     }
+
+    // MatToolbars globally 48px
+    // Adding this to body will make all mat-toolbars 48px in height
+    &[dense='mat-toolbars'] {
+      .mat-toolbar-row,
+      .mat-toolbar-single-row {
+        height: 48px;
+      }
+    }
+    // Dense toolbars
+    // * Setting this on wil propagate to all mat-toolbars
+    td-layout,
+    td-layout-nav,
+    td-layout-manage-list {
+      &[dense='mat-toolbars'] {
```

Comment:
Shouldnt this be just `[dense]`?

Reply:
Now that i think about it.. maybe dense should be an attribute class of layouts to target only things inside the component and it doesnt cascade.

Reply:
Yeah I agree that this should probably go inside Covalent

Reply:
Did a super quick pass here: https://github.com/Teradata/covalent/pull/1663